services/greasyfork/greasyfork-base.js

```diff
@@ -16,9 +16,17 @@ export default class BaseGreasyForkService extends BaseJsonService {
   static defaultBadgeData = { label: 'greasy fork' }
 
   async fetch({ scriptId }) {
-    return this._requestJson({
-      schema,
-      url: `https://greasyfork.org/scripts/${scriptId}.json`,
-    })
+    try {
+      return await this._requestJson({
+        schema,
+        url: `https://greasyfork.org/scripts/${scriptId}.json`,
+      })
+    } catch (e) {
+      if (e.message !== 'Not Found') throw e
```

Comment:
Could we check for the exception type using `instanceof` so it will still work if we customise the not found error.

e.g:

```js
import { NotFound } from '../index.js'

...

if (!(e instanceof NotFound)) throw e
```

Reply:
okay